### PR TITLE
Reduce @viewport to main feature; mark removal/deprecation

### PR DESCRIPTION
**Pull request by rachelandrew**
Updating BCD for deprecated `@viewport` as described in https://github.com/mdn/sprints/issues/3404#issuecomment-712038273

### CODE REVIEW

**Comment by Elchi3** [comment]
I think reducing this to just the `@viewport` main feature is fine even if the sub features don't yet meet the removal requirements. (not yet 2 years gone per https://github.com/mdn/browser-compat-data/blob/master/docs/data-guidelines.md#removal-of-irrelevant-features)

The only blocking thing would be to do something with the MDN sub pages like
https://developer.mozilla.org/en-US/docs/Web/CSS/@viewport/max-width

Should we redirect these to https://developer.mozilla.org/en-US/docs/Web/CSS/@viewport?

**Comment by rachelandrew** [comment]
> Should we redirect these to https://developer.mozilla.org/en-US/docs/Web/CSS/@viewport?

that would make sense to me, I don't think I can redirect or I would do it. I can move pages but that's all.

**Comment by Elchi3** [comment]
I've put the redirects in place for all sub pages of `@viewport` (there is a button in ckeditor, maybe it requires permissions?)

See https://wiki.developer.mozilla.org/en-US/docs/Web/CSS/@viewport/zoom$compare?locale=en-US&to=1655328&from=1645709 for how the change looks like.

File: css/at-rules/viewport.json [hunk 1]
```diff
@@ -117,719 +117,7 @@
           "status": {
             "experimental": false,
             "standard_track": true,
```
Comment (Elchi3) [inline css/at-rules/viewport.json:119]: I believe this should be `false` now as it is no longer in a spec?

```suggestion
            "standard_track": false,
```

**Review by Elchi3 — CHANGES_REQUESTED** [review]
It would be nice to also set the removal versions in Edge and Operas to signal that this is really gone, but we can also do that in a follow-up. 

A question about standards track but otherwise LGTM.

**Review by Elchi3 — APPROVED** [review]
Thanks Rachel! :+1:

**Pull request merged**